src/Plugin/AddMedia.php

```diff
@@ -66,8 +66,21 @@ public function execute(Bag $bag, $bag_temp_dir, $nid, $node_json, $token = NULL
                             in_array($term['url'], $this->settings['drupal_media_tags'])) {
                         if (isset($media['field_media_image'])) {
                             $file_url = $media['field_media_image'][0]['url'];
-                        } else {
+                        if (isset($media['field_media_file'])) {
```

Comment:
I had change the condition to the following to prevent a syntax error and test the presence of the `url` key. 
```
} else if (isset($media['field_media_file'][0]['url'])) {
```